Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -472,6 +480,45 @@ public extension UIView {
         CATransaction.commit()
     }
 
+    /// SwifterSwift: Apply Gradient Colors.
+    ///
+    ///     view.applyGradient(
+    ///         colors: [UIColor.red.cgColor, UIColor.blue.cgColor],
```

Comment:
If you use my suggestions, this can be simplified
```suggestion
    ///         colors: [.red, .blue],
```